---
uint64[] ids
string[] names
bool success